code/datums/traits/negative.dm

```diff
@@ -535,3 +535,13 @@
 	gain_text = "<span class='danger'>There's a lot on your mind right now.</span>"
 	lose_text = "<span class='notice'>Your mind finally feels calm.</span>"
 	medical_record_text = "Patient's mind is in a vulnerable state, and cannot recover from traumatic events."
+
+/datum/quirk/diabetic
+	name = "Diabetic"
+	desc = "You have diabetes type 1, better watch your sugars!"
+	value = -2
```

Comment:
literal 2 free points because 75 sugar in your blood will never happen unless you do it deliberately and even then it just knocks you out until the sugar leaves your blood

Reply:
look a little further down and you'll see the perk makes the overdose threshold 15, not 75. 75 is for everyone else

Reply:
owo u right